### DISCUSSION TITLE
Termux root@localhost:/usr/games/xash#

### DISCUSSION THREAD

**Discussion by DarkPhonk** [termux/termux-packages#15840] [General]
????
![Screenshot_2023-03-27-14-47-20-171_com.termux.jpg](https://user-images.githubusercontent.com/116999953/227933015-38a30de2-752a-4452-97df-29136464134b.jpg)

**Comment by sylirre** [comment]
I will close this because the amount of information provided doesn't make sense for further discussion.

I see there a crash message from QEMU (user-mode?), however you didn't provided a description for:

* What you are trying to run.
* The environment. Is that is a PRoot environment?
* Configuration details.
* Steps to reproduce the issue.

All `proot` issues should go to https://github.com/termux/proot repository.

*Do not report bugs in GitHub Discussions (https://github.com/termux/termux-packages/discussions). File an issue ticket instead.*
